### PR TITLE
Adds V1 additions to Page components and Global elements sections of the manual

### PR DESCRIPTION
**Pull request by ajbush**
This adds the following new pages

**Under** "**Global elements**"
- Browse page
- Document detail page
- Filterable list pages
- Item introduction
- Landing page
- Learn page
- Sublanding page
- Text introductions

**Under** "**Page components**"
- 25/75 image and text
- 50/50 image and text
- Buttons
- Contact information
- Email sign-up
- Filterable list control panel
- Full width text
- Half width link blob
- Sidebar/prefooter
- Wells

http://ajbush.github.io/design-manual/

### CODE REVIEW

**Comment by ajbush** [comment]
Tagging @Scotchester and @jimmynotjim to take a look and see if we're good on this.

**Comment by schaferjh** [comment]
This is so exciting! 💥

**Comment by Scotchester** [comment]
There's way too much code here to really be reviewable, but I took a quick perusal through all the pages on AJ's fork and it looks pretty good to me! We can fix any lingering issues as we discover them. :+1:

File: _config.yml [hunk 1]
```diff
@@ -150,18 +198,38 @@ secondarynav:
...
   category: Page components
-# - text: Tables
-#  url: page-components/tables.html
-#  internal: true
-#  category: Page components
+- text: Sidebar/prefooter
+  url: page-components/sidebar-prefooter.html
+  internal: true
+  category: Page components
+- text: Tables
```
Comment (Scotchester) [inline _config.yml]: Actually, I have one request for this PR before merging: Please re-comment out the Tables page in this navigation config. (Comment out lines 225–228.)

The content for this page is still being finished up by @niqjohnson & co.

Comment (ajbush) [inline _config.yml]: Cool, updated!

**Comment by jimmynotjim** [comment]
I noticed there are some broken anchor links at the top of the doc here => http://ajbush.github.io/design-manual/global-elements/text-introductions.html#style Double check all your anchor links are working as expected.

**Comment by jimmynotjim** [comment]
Other than the above link, everything looks good. 👍 from me once those links are fixed.

**Comment by ajbush** [comment]
Updated the anchor links like @jimmynotjim mentioned. There were a couple issues with missing hyphens, etc.

**Comment by Scotchester** [comment]
Lookin' good; thanks AJ!

**Pull request merged**